requirements/constraints.txt

```diff
@@ -11,3 +11,6 @@
 
 # Latest previous version has issues with doc8 because of conflicting docutils constraints
 sphinx-book-theme==0.4.0rc1
+
+# Temporary solution since this version raises RecursionError for test_generate_avro_schemas.py
```

Comment:
[curious] do we have any plans to every so often go test removing the constrains in this list so as to not make temporary fixes live forever?
Not a blocker by any means.

Reply:
I'm planning on waiting for the next astroid release to see if it's solved since the error it's from the package itself. In the meantime, I'll look over the code to figure out why it's failing with only that package. I'll include this removal plan in the in-line comment. Thansk!